.github/workflows/scripts/slack_gem_notifications/notifications_script.rb

```diff
@@ -0,0 +1,13 @@
+# encoding: utf-8
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+
+# This script runs on a 24 hour cycle via ci_cron.yml and sends slack updates for new gem version releases.
+
+require_relative 'notifications_methods'
+
+# TODO: Add support for comparing latest and previous ruby versions
+
+WATCHED_GEMS = ARGV
+
+check_for_updates(watched_gems)
```

Comment:
There's a constant called `WATCHED_GEMS` (all caps) and then a variable being passed to `check_for_updates` called `watched_gems` (all lowercase). Is this a mismatch? If so, maybe you can get rid of the constant and just do `check_for_udpates(ARGV)`.